docs/images/siddhi-logo-w.svg

```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Please change the license header to 2018. Check other places as well

Reply:
Resolved with the commit 7e05e862337f827854551f7cdea54c88c72623e4.